src/backend/utils/systeminfo/index.ts

```diff
@@ -137,6 +143,8 @@ The current system is${info.steamDeckInfo.isDeck ? '' : ' not'} a Steam Deck${
       : ''
   }
 We are${info.isFlatpak ? '' : ' not'} running inside a Flatpak container
+We are${info.isNixPkg ? '' : ' not'} running the community nix package
```

Comment:
Adding to my last comment: Even if we do keep the bool in `SystemInformation`, I don't think it's necessary to be printed here. If you want to make it obvious in the log that we're on NixOS, that's already visible in the "OS" entry

Reply:
Knowing that we are in NixOS does not tell us, on its own, that we are running the nix package, because we could be running the AppImage.  How about instead log a message like we do for Flatpak, but for AppImage?